spec/models/appointment_spec.rb

```diff
@@ -437,14 +437,40 @@
         end
       end
 
-      context 'appointment has been rescheduled' do
-        it 'needs another reminder' do
-          appointment.update!(
-            start_at: BusinessDays.from_now(20),
-            end_at:   BusinessDays.from_now(20) + 1.hour
-          )
+      context 'has already had a reminder' do
```

Comment:
I'll revisit this. I felt the same way but wanted to get it out the door, but it's the right thing to do.